docker/volumes/db/init/01-auth-schema.sql

```diff
@@ -88,21 +88,57 @@ VALUES  ('20171026211738'),
         ('20180108183307'),
         ('20180119214651'),
         ('20180125194653');
-		
--- Gets the User ID from the request cookie
-create or replace function auth.uid() returns uuid as $$
-  select nullif(current_setting('request.jwt.claim.sub', true), '')::uuid;
-$$ language sql stable;
-
--- Gets the User ID from the request cookie
-create or replace function auth.role() returns text as $$
-  select nullif(current_setting('request.jwt.claim.role', true), '')::text;
-$$ language sql stable;
-
--- Gets the User email
-create or replace function auth.email() returns text as $$
-  select nullif(current_setting('request.jwt.claim.email', true), '')::text;
-$$ language sql stable;
+
+create or replace function auth.uid() 
+returns uuid 
+language plpgsql stable
+as $$
+declare
+    sub text := current_setting('request.jwt.claim.sub', true);
+    claims text := current_setting('request.jwt.claims', true);
+    id uuid;
+begin
+  select case 
+  	when sub is not null then sub::uuid
```

Comment:
We need a tweak here to handle empty string to null coercion because of this behavior:

```sql
begin
        -- set a config value to NULL
	select set_config('request.jwt.null_to_empty_str', null, true);
	
	select current_setting('request.jwt.null_to_empty_str', true);
	-- Result: '' vs NULL
rollback
```